### PR TITLE
Fix deprecation: Use `getThrowable` when possible

### PR DESCRIPTION
**Pull request by ruudk**
Fixes https://github.com/ekino/EkinoNewRelicBundle/issues/237

### CODE REVIEW

File: Listener/ExceptionListener.php [hunk 1]
```diff
@@ -40,9 +40,14 @@ public static function getSubscribedEvents(): array
 
     public function onKernelException(GetResponseForExceptionEvent $event): void
```
Comment (jderusse) [inline Listener/ExceptionListener.php]: this class does not exists anymore in 5.0 .

I'm trying to fix all this changes in #239

**Comment by ruudk** [comment]
Thanks, I will subscribe to the other issue. Feel free to close this if not needed.

**Comment by jderusse** [comment]
PR #239 is green. a lot of things to fix... would you mind to have a look @ruudk ?

**Comment by jderusse** [comment]
fixed by #239

**Pull request closed without merging**